src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
```suggestion
  return {
```
you may return it immediately instead of creating variable